### DISCUSSION TITLE
how to add urls with wildcard to whitelist of project, or how to add urls to whitelist using apis

### DISCUSSION THREAD

**Discussion by smartdev0530** [Web3Auth/Web3Auth#973] [Help]
I want to add several urls to whitelist of projects, and I wonder if there's a way to do it automatically.
Can I use wild carded urls, or is there any api to add urls to whitelist?

**Comment by tanguyenvn** [comment]
So far we don't support wildcards when whitelisting Projects.
cc @chaitanyapotti Do we have any plan to support wildcards?

**Comment by chaitanyapotti** [comment]
Duplicate of https://github.com/orgs/Web3Auth/discussions/351

**Comment by marcus-sa** [comment]
@chaitanyapotti are wildcards ever gonna be supported? We need it for preview deployments...
Having to generate a signature for every preview environment and injecting it is simply ridiculous.

**Reply by marcus-sa** [reply]
@chaitanyapotti any updates..? Is Web3Auth even actively maintained..?

**Reply by rpedroni** [reply]
This still doesn't exist?

**Reply by marcus-sa** [reply]
@rpedroni there are better maintained alternatives to Web3Auth out there. 
Honestly, the fact that this is a paid product and they don't even respond is ridiculous.
